packages/backend/src/models/ProjectModel/ProjectModel.ts

```diff
@@ -1053,6 +1053,10 @@ export class ProjectModel {
                     (a, b) => a.name === b.name,
                 );
 
+                if (uniqueExplores.length <= 0) {
+                    throw new ParameterError('No explores to save');
```

Comment:
ParseError or NotFoundError

Reply:
Parse error because it relates to the arguments rather than database results.